modules/lambda_function/lambda-function.md

```diff
@@ -20,7 +20,8 @@ much more setup. Currently, it also handles:
 * Allowing the user to pass in IAM policies to give to the lambda
 * Setting up security group and network location if you have the `aws-base` module in your environment.
 
-tl;dr -- this will deploy a hello world lambda
+tl;dr -- Download [this zip](https://gist.github.com/juandiegopalomino/7400f836107459f3099c02e58d2d6897/raw/5c0b4cbbbf1d8a470f16c09d53e6ed68e59e06bd/baloney.zip), 
```

Comment:
Slightly better way to write this: "Simple way to try out this module: Just download this example zip file, put it in the same directory as your opta.yml and run opta apply".

Reply:
Also, it's better to put the zip file in our docs static dir (https://github.com/run-x/opta-docs/tree/main/static) - instead of a personal gist.

Reply:
We got a chicken and the egg scenario here. Once the lambda is deployed and docs first updated, we can make an update to point to a zip file in our docs.